ext/webgpu/01_webgpu.js

```diff
@@ -5302,6 +5302,18 @@ webidl.converters["GPUExtent3D"] = (V, opts) => {
     return webidl.converters["GPUExtent3DDict"](V, opts);
   }
   if (typeof V === "object") {
+    // validate GPUExtent3D when its an Array
+    if (ArrayIsArray(V)) {
+      const min = 1;
+      const max = 3;
+      if (V.length < min || V.length > max) {
+        throw webidl.makeException(
+          TypeError,
+          `A sequence of number used as a GPUExtent3D must have between ${min} and ${max} elements.`,
+          opts,
+        );
```

Comment:
This error message picks from chromium.

https://source.chromium.org/chromium/chromium/src/+/refs/tags/122.0.6261.112:third_party/blink/renderer/modules/webgpu/dawn_conversions.cc;l=84-86

Reply:
BTW, should I implement validations for other typedef of `GPUColor`, `GPUOrigin2D` and `GPUOrigin3D`?

Reply:
yes, that would be great